blocks/header/header.css

```diff
@@ -251,4 +628,618 @@ outline: none;
   height: 100%;
   width: 100%;
   top: 0;
-}
\ No newline at end of file
+}
+
+@media (width >=768px) and (width<= 1023px) {
+  header .nav-wrapper{
+    overflow-y: visible;
+  }
+
+  header .nav-menu {
+overflow-y: scroll;
+
+    /* min-width: 100vw; */
+    background-image: url('../../icons/swoosh/type4/default-tab.svg');
+    min-height: 100vh;
+    min-width: 100%;
+  }
+
+  header .nav-menu-overlay{
+    min-width: 100vw;
+    overflow: hidden scroll;
+
+    
+  }
+
+header .nav-menu-overlay .nav-menu {
+  background-size:70% 100%;
+
+  /* background-position: contain; */
+  
```

Comment:
Please remove. Format the whole document if possible.

Reply:
Done